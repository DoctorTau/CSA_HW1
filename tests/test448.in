62
7223 -291 912 3530 -1689 9193 9376 5410 6041 -3991 -4350 8244 -1922 6700 2095 -4970 6551 -5326 8187 4971 4880 -391 7251 9405 -827 -6463 2425 -445 1949 -7700 7076 9537 4503 3702 243 6263 3698 -1190 -741 1586 9300 -8013 7954 2796 1287 3132 -443 7056 -1891 2351 -139 790 8715 8717 -6178 573 -6926 -1502 -7271 3619 5020 1141 