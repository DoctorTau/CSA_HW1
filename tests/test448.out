-4350 -1922 -4970 -5326 -7700 -1190 -6178 -6926 -1502 